target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts

```diff
@@ -54,3 +65,7 @@
 		ieee80211-freq-limit = <5000000 6000000>;
 	};
 };
+
+&sdhci {
```

Comment:
As in PR #1500, unless the bug is fixed, blindly enabling SD will render ethernet ports unusable